src/main/java/io/personium/core/rs/odata/AbstractODataResource.java

```diff
@@ -191,21 +191,16 @@ private MediaType decideOutputFormatFromQueryValue(String format) {
      * @return output format ("application / json" or "application / atom + xml")
      */
     private MediaType decideOutputFormatFromHeaderValues(String acceptHeaderValue) {
-        MediaType mediaType = null;
-        StringTokenizer st = new StringTokenizer(acceptHeaderValue, ",");
-        while (st.hasMoreTokens()) {
-            String accept = truncateAfterSemicolon(st.nextToken());
-            if (isAcceptXml(accept)) {
-                mediaType = MediaType.APPLICATION_ATOM_XML_TYPE;
-            } else if (isAcceptJson(accept)) {
-                if (mediaType == null) {
-                    mediaType = MediaType.APPLICATION_JSON_TYPE;
-                }
-            } else {
-                throw PersoniumCoreException.OData.UNSUPPORTED_MEDIA_TYPE.params(acceptHeaderValue);
-            }
+        String[] types = Stream.of(acceptHeaderValue.split(","))
```

Comment:
This should be `acceptHeaderValue.split("[ \t]*,[ \t]*")`.
https://tools.ietf.org/html/rfc7231#section-5.3.2
https://tools.ietf.org/html/rfc7230#section-7
https://tools.ietf.org/html/rfc7230#section-3.2.3

Reply:
I modified it. If you have time, could you review it?

Reply:
I found no problems.